src/SpendPermissionManager.sol

```diff
@@ -478,8 +499,18 @@ contract SpendPermissionManager is EIP712 {
 
     /// @notice Approve spend permission.
     ///
+    /// @dev Emits a {SpendPermissionApproved} event if the spend permission is newly approved and not already revoked.
```

Comment:
nit: thoughts on this syntax instead? Would match how we reference specific in-code items in other comments
```suggestion
    /// @dev Emits a `SpendPermissionApproved` event if the spend permission is newly approved and not already revoked.
```